Fenrir.Tests/Fenrir.Tests.fsproj

```diff
@@ -12,7 +12,20 @@
         <Compile Include="TestUtils.fs" />
         <Compile Include="CommandTests.fs" />
         <Compile Include="RefsTests.fs" />
+        <Compile Include="PackTests.fs" />
         <Compile Include="Program.fs" />
+        <Content Update="Data\pack-07a2498901f9e70c9056ae89d11aba80b3402616.idx">
+          <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </Content>
+        <Content Update="Data\pack-07a2498901f9e70c9056ae89d11aba80b3402616.pack">
+          <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </Content>
+        <Content Update="Data\objects\pack\pack-07a2498901f9e70c9056ae89d11aba80b3402616.pack">
+          <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </Content>
+        <Content Update="Data\objects\pack\pack-07a2498901f9e70c9056ae89d11aba80b3402616.idx">
+          <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </Content>
```

Comment:
Adding these `<Content>` elements shouldn't be necessary, because we already have everything from `Data` folder included with `<Content Include="Data\**\*" />`.

Reply:
This stuff was generated by IDE.
But I'll remove it anyway.